Gemfile

```diff
@@ -6,16 +6,18 @@ git_source(:github) do |repo_name|
 end
 
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.1.4'
-# Use sqlite3 as the database for Active Record
```

Comment:
I guess these could be useful for newbies like me, maybe they can stay?

Reply:
k ill restore default commments